Formula/zork.rb

```diff
@@ -0,0 +1,30 @@
+class Zork < Formula
+  desc "The Dungeon modified from FORTRAN to C"
+  homepage "https://github.com/devshane/zork"
+  url "https://github.com/devshane/zork/archive/v1.0.0.tar.gz"
+  sha256 "ceedb436cf552bd8ca39abd3e85bae986c3f5df6f31c1bf2806d1b4c8509da67"
+  head "https://github.com/devshane/zork.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "make", "LIBDIR=#{libexec}", "BINDIR=#{bin}"
+    bin.mkpath
+    libexec.mkpath
```

Comment:
Sure thing, I can submit a PR to the upstream Makefile too. Just to make sure I understand correctly, `make`  should handle ensuring these directories exist or should `make install` handle that bit?